signingscript/sign.py

```diff
@@ -289,7 +290,7 @@ def _should_sign_windows(filename):
 
 # _should_sign_widevine {{{1
 def _should_sign_widevine(filename):
-    """Return (True, blessed) if filename should be signed."""
+    """Return signing format if filename should be signed."""
```

Comment:
Should we explicitly return `None` when a file doesn't match? It would make the intent clearer, IMO